main.go

```diff
@@ -301,6 +301,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
Why do you need the option to disable the webhook?

Reply:
Oh, it's likely for when it's not run in the cluster.

Could you please create a new flag/argument instead of an environment variable for this?